_pages/userland/index.html

```diff
@@ -27,35 +27,30 @@
       available in Electron "core".
     </p>
 
-    <h2>Reports</h2>
+    <p>
+      We analyzed metadata from 15,000 public GitHub repositories
+      that depend on <code>electron</code> or <code>electron-prebuilt</code>. To collect
+      this data, we used the GitHub API,
+      the <a href="https://libraries.io/api">Libraries.io API</a>,
+      and the npm registry.
+      All of the tools used to collect this data are open-source. To find out more
+      about how this data is collected, see <a href="https://github.com/electron/electron-userland-reports">electron/electron-userland-reports</a>
+    </p>
 
-    <ul>
+    <ul class="report-list">
     {% for thing in site.data.userland %}
       {% for report in thing %}
         {% if report.title %}
           <li>
-            <a href="{{site.baseurl}}/userland/{{report.slug}}">{{report.title}}</a>
-            - {{report.description}}
+            <a href="{{site.baseurl}}/userland/{{report.slug}}" class="inner">
```

Comment:
@zeke Added the icons under `images/userland/`. The markup might be:

```html
<img src="{{site.baseurl}}/images/userland/{{report.slug}}.svg" alt="{{report.title}} icon">
```

We could also add a fixed size to the images, so that the page doesn't jump after loading them.

```
.report-list img {
  display: inline-block;
  width: 120px;
  height: 60px;
}
```

And it might also need some bottom margin.